manual/docs/front-end-testing.md

```diff
@@ -0,0 +1,38 @@
+# Front-End Testing
```

Comment:
The front-end team actively maintains their guides at https://github.com/cfpb/front-end. I really think we're better off just linking to them there.